packages/notification/src/types.ts

```diff
@@ -7,11 +7,15 @@ export type ToastNotification = {
   type: NotificationType
   title: string
   message: React.ReactNode
-  autohide: boolean
   autohideDelay?: "short" | "long"
   onHide?: () => void
   automationId?: string
   persistent?: boolean
+  /**
+   * **Deprecated:**
+   * @deprecated
```

Comment:
Probably want to put some info in here

```suggestion
   * @deprecated Automatic hiding is an accessibility anti-pattern, violating WCAG '2.2 Enough Time' criteria. This prop will be removed in an upcoming breaking change.
```